AtTools/AtLineChargeModel.cxx

```diff
@@ -49,6 +49,10 @@ XYZPoint AtLineChargeModel::ApplySpaceCharge(const XYZPoint &reverseInputPositio
    auto delZ = input.Z();         // in mm
    double dist2 = getDist2(delZ); // in mm;
    double newRho = sqrt(input.Rho() * input.Rho() - dist2);
+   if (!(newRho > 0)) {

```

Comment:
Easier to understand if code is:
```
if(newRho < 0) newRho = 0;
```

It took me longer than it should to decode this lol